utils.sh

```diff
@@ -270,7 +270,12 @@ function master_node_to_tf() {
     local_gb=$(master_node_val ${master_idx} "properties.local_gb")
     cpu_arch=$(master_node_val ${master_idx} "properties.cpu_arch")
 
-    ipmi_port=$(master_node_val ${master_idx} "driver_info.ipmi_port")
```

Comment:
Try this, if it works you can remove the other lines
`ipmi_port=$(master_node_val ${master_idx} "driver_info.ipmi_port // \"\"")`

Reply:
This returns as empty value.
The thing is that later in this file we have:

  driver_info {
    "ipmi_port" = 

The value should be some valid port and not empty/null.

I replace the entire line with '${ipmi_port}'

Reply:
Sorry, I didn't read the rest of the code correctly, I was trying to avoid using $NODES_PLATFORM as we could have BM nodes with a port (although its unlikely) or libvirt nodes without a port. How about something like this

```
ipmi_port=$(master_node_val ${master_idx} "driver_info.ipmi_port // \"\"")
if [ -n "$ipmi_port" ]; then
    ipmi_port="\"ipmi_port\"=      \"${ipmi_port}\""
fi

```